packages/calcite-components/src/components/popover/popover.tsx

```diff
@@ -522,6 +529,22 @@ export class Popover
     this.reposition(true);
   };
 
+  private clickOutsideDeactivates = (event: MouseEvent): boolean => {
```

Comment:
Can you confirm callbacks passed to `connectFocusTrap` options need to be `this`-bound?

Reply:
Confirmed. Fails tests without.